template/template.go

```diff
@@ -112,6 +112,11 @@ type executer struct {
 	template  *Template
 	w         io.Writer
 	variables map[string]any
+	errs      []error
```

Comment:
not needed?

Reply:
It is needed to collect possible errors from several selectors